app/content/deployment-options/custom-deployment/operations.md

```diff
@@ -104,3 +104,66 @@ source .env && docker run --rm \
 ```
 
 
+### Upgrading to a new version
+
+CARTO On-Premises can be upgraded with just a few steps. To proceed, you should ask Support team for your upgrade files.
+
+First you'll need to perform a backup of your data:
+
+1. Stop Docker containers
+```bash
+docker-compose stop
+```
+2. Create a backup of the current version's logs
+```bash
+docker-compose logs -t > backup.log
+```
+3. Copy the current configuration files.
+```bash
+cp -r /previous/installation/folder /carto-backup
+```
+{{% bannerNote title="WARNING"%}}
+You'll need to substitude */previous/installation/folder* and */carto-backup* for the actual origin and destination of the configuration files.
```

Comment:
I think we should use "replace" or "substitute" (with a t)  instead of "substitude"